pod.go

```diff
@@ -269,6 +269,12 @@ type PodConfig struct {
 	// This list can be empty and populated by adding containers
 	// to the Pod a posteriori.
 	Containers []ContainerConfig
+
+	// Annotations is a reserved field and can be filled with various
```

Comment:
`reserved` fields usually refer to unused and undefined block of a frame or structure. This one is well defined and refers to e.g. OCI annotations. So we should not use that word here.